server.ts

```diff
@@ -19,29 +17,41 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.set("view engine", "ejs");
 
 app.get("/", (req, res) => {
-	res.render("index", { txid: null, error: null });
+    res.render("index", { txid: null, error: null });
 });
 
 app.post("/", async (req, res) => {
     const address = req.body.address;
 
     if (address === process.env.DISTRIBUTE_SECRET!) {
-
         try {
-            await slpFaucet.evenlyDistributeTokens(process.env.TOKENID!);
+            if (process.env.NFT === 'yes') {
```

Comment:
Lets keep this part of the server logic the same as master.  Token type logic should be handled in slpfaucet.ts.